pom.xml

```diff
@@ -44,7 +44,7 @@
       <dependency>
         <groupId>io.vertx</groupId>
         <artifactId>vertx-stack-depchain</artifactId>
-        <version>3.4.0</version>
+        <version>${vertx.version}</version>
```

Comment:
Given this value is only used in one place, why extract it to a property?

Reply:
Good point. I did this when I first started work on the module, assuming there would be more places using the version number in the pomfile than there turned out to be.  I'll revert.